hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -721,6 +721,8 @@ public class HoodieWriteConfig extends HoodieConfig {
           + "The class must be a subclass of `org.apache.hudi.callback.HoodieClientInitCallback`."
           + "By default, no Hudi client init callback is executed.");
 
+  public static final String WRITE_PREPPED_MERGE_KEY = "_hoodie.datasource.merge.into.prepped";
```

Comment:
can we add java docs calling out the purpose of this

Reply:
sorry. lets name this
_hoodie.spark.sql.merge.into.prepped

Reply:
lets name the variable 
SPARK_SQL_MERGE_INTO_PREPPED

Reply:
Fixed.